packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -345,7 +346,12 @@ export class DocumentDeltaConnection
 			throw new UsageError("Sending signals to specific client ids is not supported.");
 		}
 
-		this.emitMessages("submitSignal", [[content]]);
+		const signal: ISentSignalMessage = {
+			content,
+			targetClientId,
+		};
+
+		this.emitMessages("submitSignal", [signal]);
```

Comment:
I don't think we can blanket submit `ISentSignalMessage[]`. The shape of the received content won't be correct if the service doesn't support v2.
The targetClientId and v2 check above is incorrect per chat https://teams.microsoft.com/l/message/19:0f94bd601565474bab29154a77a4528f@thread.v2/1712080563331?context=%7B%22contextType%22%3A%22chat%22%7D -- more visible discussion at https://github.com/microsoft/FluidFramework/pull/17729/files#r1533281026